.github/workflows/build_pr_documentation.yml

```diff
@@ -15,4 +15,4 @@ jobs:
       pr_number: ${{ github.event.number }}
       install_libgl1: true
       package: diffusers
-      languages: en ko zh
+      languages: en ko zh jp
```

Comment:
```suggestion
      languages: en ko zh ja
```

Reply:
Nice catch! And I fixed the folder name to be ja too